Makefile

```diff
@@ -44,6 +44,7 @@ docker-bootstrap: clean
 
 venv: 
 	virtualenv -p python2 $(ENV)
+	$(ENV)/bin/pip install -U pip
```

Comment:
Update also `setuptools`, is something that brought us some issues being old after some changes they did last year.

Reply:
Thanks! Adding.